README.md

```diff
@@ -247,30 +247,30 @@ OR simply
 re.reload("*");
 ```
 
-* Get the work order in JSON or Byte Array:
+* Get the work order in JSON or byte array:
 
 ```java
 JsonObject jo = re.toJSON();
 byte[] joBytes = re.toJSONBytes();
 ```
 
 ### 3.1.3 Traverse the Work Orders 
-In some case, there is a need to traverse the some or all work orders. There are some helpful API in  Maximo Rest Client.
+In some case, you might need to traverse some or all work orders. There are some helpful API in the Maximo REST Client.
 
-* Connect to the Maximo, 
+* Connect to the Maximo Asset Management: 
 
 ```java
 MaximoConnector mc = new MaximoConnector(new Options().user("maxadmin").password("maxadmin").lean(true).auth("maxauth").host("host").port(7001));
 mc.connect();
 ```
 
-* Get a work order set from Maximo server.
+* Get a work order set from the Maximo server:
 
 ```java
 ResourceSet rs = mc.resourceSet("mxwodetail").pageSize(10).
 ```
  
-* Travel through the workorders in current page.
+* Travel through the work orders that are on the current page:
```

Comment:
line 273
clarifying language - please make sure that I've not inadvertently changed the meaning